news/2024/2024-09-11-osumania-4k-world-cup-2024-quarterfinals-recap.md

```diff
@@ -0,0 +1,117 @@
+
+## Quarterfinals results
+
+Don't feel like going through the wiki page for the full results? We got you covered.
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| **China** ::{ flag=CN }:: | **6** | 1 | ::{ flag=ID }:: Indonesia |
+| **Japan** ::{ flag=JP }:: | **6** | 1 | ::{ flag=ES }:: Spain |
+| **United Kingdom** ::{ flag=GB }:: | **6** | 2 | ::{ flag=AU }:: Australia |
+| **Hong Kong** ::{ flag=HK }:: | **6** | 2 | ::{ flag=SE }:: Sweden |
+| **Russian Federation** ::{ flag=RU }:: | **6** | 0 | ::{ flag=FR }:: France |
+| **Canada** ::{ flag=CA }:: | **6** | 2 | ::{ flag=AR }:: Argentina |
+| **Brazil** ::{ flag=BR }:: | **6** | 5 | ::{ flag=MX }:: Mexico |
+| Philippines ::{ flag=PH }:: | 2 | **6** | ::{ flag=CL }:: **Chile** |
+| **United States** ::{ flag=US }:: | **6** | 0 | ::{ flag=MY }:: Malaysia |
+| **Taiwan** ::{ flag=TW }:: | **6** | 2 | ::{ flag=NZ }:: New Zealand |
+| **South Korea** ::{ flag=KR }:: | **6** | 1 | ::{ flag=VN }:: Vietnam |
+| **Russian Federation** ::{ flag=RU }:: | **6** | 2 | ::{ flag=TW }:: Taiwan |
+| **Thailand** ::{ flag=TH }:: | **6** | 1 | ::{ flag=SG }:: Singapore |
+| Brazil ::{ flag=BR }:: | 0 | **6** | ::{ flag=JP }:: **Japan** |
+| United Kingdom ::{ flag=GB }:: | 5 | **6** | ::{ flag=HK }:: **Hong Kong** |
+| **China** ::{ flag=CN }:: | **6** | 1 | ::{ flag=CA }:: Canada |
+
+## Match highlights
+
+### ::{ flag=BR }:: Brazil vs. ::{ flag=MX }:: Mexico
+
+[**You can view the match information here!**](https://osu.ppy.sh/community/matches/115406325)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2245307636&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Brazil and Mexico is a match-up viewers were excited for. Both teams are generally all-rounders, meaning that this match was anyone's game, especially with each team's seeding being #12 and #18 respectively. If you are familiar with the Brazil team from previous years, you may know that they are former winners of the Mania World Cup back in 2020 and 2021. Mexico on the other hand are more or less newcomers who made their first appearance in the main brackets in the 2023 rendition of the world cup, so if Mexico wanted to prove that they could take down any fight, this was the match that they had to win.
```

Comment:
```suggestion
Brazil and Mexico is a matchup viewers were excited for, as it could've been anyone's game for the closely seeded teams, both featuring all-rounders. Regular spectators may recall Brazil's past victories in the osu!mania 4K World Cup back in 2020 and 2021.

Team Mexico on the other hand is composed more or less of newcomers who made their first appearance in the main brackets in MWC 4K 2023, so if Mexico wanted to prove that they could take down any fight, this was the match that they had to win.
```